README.md

```diff
@@ -21,6 +21,33 @@ All our assets can be found in the [*Assets*-Folder](./Assets). This includes:
 
 All assets were created by [Fabian Huettenhoff](https://twitter.com/zuqbu). A **big** Thank you for this!
 
+## Raffle
+
+From time to time we do a raffle.
+The winnings will be (in most of the times) provided by various sponsors.
+Winnings we had in the past were
+
+* power banks (e.g. to recharge your phone)
+* programing language mascots (e.g. [ElePHPant](https://secure.php.net/elephpant.php))
+* conference tickets
+
+To select a winner, we assume that everyone signed up in the meetup event page.
+We open the event page, open the developer console of the browser and use one of the following javascript snippets to select a winner:
+
+"going" only
+```js
```

Comment:
let's use the gist directly. You can use the JS embed and put it in here, like described here:
https://stackoverflow.com/questions/11622509/github-how-to-embed-a-gist-into-readme-md

Reply:
Not possible in a README / with Github Markdown. See https://github.com/github/markup/issues/199 or the comments in your Stack Overflow